README.md

```diff
@@ -1,48 +1,23 @@
 # cime
 Common Infrastructure for Modeling the Earth
 
-**cime** (pronounced: seem) represents the infrastructure code for the 
+ **cime** (pronounced: seem)providess scripts for configuration, build, and testing of
```

Comment:
I think that the right content here is to put in a more comprehensive statement - like what is contained at the top of the User's guide:
CIME, pronounced “SEAM”, contains the support scripts (configure, build, run, test), data models, essential utility libraries, a “main” and other tools that are needed to build a single-executable coupled Earth System Model. CIME is available in a stand-alone package that can be compiled and tested without active prognostic components but is typically included in the source of a climate model. CIME does not contain: any active components, any intra-component coupling capability (such as atmosphere physics-dynamics coupling).

Reply:
Sure I'll make that change.

Reply:
You did not make the above change . Could you please do that and I can then accept the PR.